packages/modeling/src/operations/transforms/index.js

```diff
@@ -6,6 +6,11 @@
  * const { center, rotateX, translate } = require('@jscad/modeling').transforms
  */
 module.exports = {
+  align: require('./align').align,
+  alignX: require('./align').alignX,
```

Comment:
I didn't see these 'aliases' defined in align.js

Reply:
yeah, I started on them, and it didn't feel quite right separating out the axes.  Thoughts?
```alignX({mode: "lower", alignTo: 0, grouped:true}, ...objects)```
Worth it or not?

Reply:
Nope

Reply:
removed